config/management/operational/src/command.rs

```diff
@@ -31,7 +31,11 @@ pub enum Command {
     #[structopt(about = "Set the waypoint in the validator storage")]
     InsertWaypoint(libra_management::waypoint::InsertWaypoint),
     #[structopt(about = "Prints an account from the validator storage")]
-    PrintAccount(crate::account::PrintAccount),
+    PrintAccount(crate::print::PrintAccount),
+    #[structopt(about = "Prints a public key from the validator storage")]
+    PrintKey(crate::print::PrintKey),
+    #[structopt(about = "Prints a waypoint from the validator storage")]
+    PrintWaypoint(crate::print::PrintWaypoint),
```

Comment:
hahaha, you're already taking away part of my project proposal for this upcoming half... I wonder if we would rather have a generic tool for reading / writing storage rather than this?

Reply:
Hah! Indeed -- it might be good to create a more general tool. Right now I'd argue that if we want more access to vault, we could always query vault directly, but if we create a more general secure storage (not vault specific) a tool would be ideal! 😄